doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_fetch.rst

```diff
@@ -132,9 +121,13 @@ To reconfigure the sample to use the fetch model, complete the following steps:
            - suit-directive-fetch:
              - suit-send-record-failure
 
-     When the secure domain firmware processes the ``suit-install`` sequence, this sequence of directives instructs the secure domain to search for a payload with a given URI in all cache partitions.
-     If no such payload is found, the update process ends with an error.
+      The SUIT procedure attempts to use all fetch sources registered with :c:func:`suit_dfu_fetch_source_register` until one of them fetches the payload.
+      If no sources are able to fetch the payload, the update process ends with an error.
+
+      The reference SMP fetch source implementation only recognizes URIs that start with :file:`file://`.
```

Comment:
Those lines should be removed.

Reply:
Applied, and previous comment as well. Please re-review and ensure I have done so correctly.